README.md

```diff
@@ -1,6 +1,6 @@
 [![Documentation Status][badge]][docs] [![Translation Status][tbadge]][wbetranslate]
 
-# Zammad User Documentation
+# Zammad user documentation
```

Comment:
I would actually stick with capitalization style, as this is a heading. `Zammad User Documentation`